src/Symfony/Component/DomCrawler/Field/FormField.php

```diff
@@ -102,9 +102,9 @@ public function getValue()
      *
      * @param string $value The value of the field
      */
-    public function setValue($value)
+    public function setValue(?string $value)
     {
-        $this->value = (string) $value;
```

Comment:
You cannot remove the cast as passing a `null` value won't be converted to an empty string anymore.